### ISSUE TITLE
Animated BPG Images

### ISSUE THREAD

**Issue by smaragdus** [asimba/pybpgviewer#4]
@asimba 

It seems that **Simple BPG Image viewer** does not support animated BPG images, it shows such files as static images. Support for animated BPG images would be great (at least for me). You may have a look at [BPG animation tests](https://bellard.org/bpg/animation.html). Have you tested **Simple BPG Image viewer** with animated BPG images?

**Comment by asimba** [comment]
Yes. You're right. Animation doesn't support at this moment. It requires some complex modifications of original "bpgdec" or direct using of libbpg via ctypes. This may take for a while.

**Comment by smaragdus** [comment]
@asimba 

Alexey, your work is incredible! I tested **Simple BPG Image viewer** version [1.26](https://github.com/asimba/pybpgviewer/releases/tag/v1.26) and it plays animated BPG files just right. I [notified](https://www.portablefreeware.com/forums/viewtopic.php?f=4&t=24274&p=91201#p91201) The Portable Freeware community about the new release.

See an example of [an animated PNG image](https://images2.imgbox.com/2e/cd/5VIi0E8U_o.png) I recorded showing **Simple BPG Image viewer** playing an animated BPG file.

**Comment by smaragdus** [comment]
@asimba 

I would like to thank you for all your great work and for implementing all the features I suggested!
I am glad that there are developers like you!
Have a great summer!

**Comment by asimba** [comment]
You're welcome! Nothing special - just a hobby + free time. :) That mix works like antidepressant for me. Best wishes! :)
